src/api/index.ts

```diff
@@ -74,7 +72,7 @@ export async function resetVm(id: number): Promise<string> {
 }
 
 export async function getModule(id: number) {
-  return ( await api.get<Module>(`/modules/${id}`)).data;
+  return ( await api.get<Module>(`/module/${id}`)).data;
```

Comment:
Was this broken this whole time somehow or does it work for both plural and singular?

Reply:
It was like that. Plural didn't work for me in postman so if it was working before I have no clue how lol.